css/style.css

```
@@ -567,16 +862,81 @@ padding-right: 24px;*/
   background-color: var(--classic-yellow);
 }
 
-/*
+.materials a,
+.tools a,
+.machinery a,
+.discounts a,
+.delivery a {
+  font-size: 14px;
+  line-height: 18px;
+  font-family: "Cuprum", "Tahoma", sans-serif;
+  text-transform: uppercase;
+  color: var(--classic-white);
+  background: rgb(255, 255, 255, 0.3);
```

```suggestion
  background-color: rgba(255, 255, 255, 0.3);
```
никогда не используй шоткат bg, если тебе нужен только цвет. И если есть опасити, то это альфа-канал, поэтому rgba

---

Цвет надо вынести в переменную